app/views/sign_up/passwords/new.html.slim

```diff
@@ -16,6 +16,6 @@ p.mt2.mb0#password-description
 
 = render 'shared/password_accordion'
 
-= render 'shared/cancel', link: destroy_user_session_path
+= render 'shared/cancel', link: destroy_user_session_path, decorated_session: decorated_session
```

Comment:
No need to pass in the decorated_session here since it's already available to the view automatically by being defined as a `helper_method` in ApplicationController.